### PR TITLE
Fix Editor Line Number Offset

### PR DESCRIPTION
**Pull request by dagrooms52**
- Issue #217 
- Remove top margin which offset line numbers in detector editor
- Ran style tool on scss

Screenshot:
line_numbers

### CODE REVIEW

File: AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.scss [hunk 1]
```diff
@@ -239,154 +246,161 @@ input[type="submit"]:active {
...
+    /* transform: rotate(90deg); */
+    transition: .3s;
+    width: 80px;
+    height: 80px;
+    border: 2px solid;
+    /* color: pink; */
+}
+
+::ng-deep .monaco-editor .margin {
+    margin-top: 0;
```
Comment (dagrooms52) [inline AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.scss]: This is the actual change for the PR, everything else is style fixes

**Comment by stevenernst131** [comment]
Awesome!

File: AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.scss (hunk 1 above)
Comment (cindylovescoding) [reply AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.scss]: Just want to check if this also takes care of the commits history view:)

Comment (dagrooms52) [reply AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.scss]: Good catch, I hadn't checked that yet but I applied it there too

**Pull request merged**